task2.py

```diff
@@ -0,0 +1,32 @@
+"""Create unit tests for any program from previous homeworks."""
+import unittest
+
+
+def count_characters(s):
+    """Function count and return the numbers of each character in a string."""
+    dct = {}
+    for symb in s:
+        dct[symb] = dct.get(symb, 0) + 1
+    return dct
+
+
+class TestCountCharacters(unittest.TestCase):
```

Comment:
why in count characters test collection are upper lower etc?

Reply:
solved.